build.xml

```diff
@@ -155,7 +156,10 @@
     </target>
 
     <target name="phpstan" depends="tests-acceptance-build" description="Performs static analysis of PHP files using PHPStan on all packages.">
-        <exec executable="${path.phpstan.executable}" logoutput="true" passthru="true" checkreturn="true">
+        <exec executable="${path.php.executable}" logoutput="true" passthru="true" checkreturn="true">
```

Comment:
as @boris-brtan pointed out, the proper solution might be using `--memory-limit` option, see phpstan/phpstan/issues/1148

Reply:
nevertheless, i couldnt find it in options of our phpstan

Reply:
Solved differently in #716